<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string></string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="exec_button">
            <property name="text">
             <string>Почати</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>0.5</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>p</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Файли</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="1" column="1">
                 <widget class="QToolButton" name="select_input">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="line_output">
                  <property name="text">
                   <string>lab_5/output.xlsx</string>
                  </property>
                  <property name="placeholderText">
                   <string>Вихідний файл</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QToolButton" name="select_output">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="line_input">
                  <property name="text">
                   <string>lab_5/input.xlsx</string>
                  </property>
                  <property name="placeholderText">
                   <string>Input file</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Перше завдання</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableWidget" name="tablewidget_first">
             <property name="rowCount">
              <number>6</number>
             </property>
             <property name="columnCount">
              <number>12</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Друге задвдання</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tablewidget_second">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>980</width>
         <height>230</height>
        </size>
       </property>
       <property name="rowCount">
        <number>7</number>
       </property>
       <property name="columnCount">
        <number>8</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Третє завдання</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tablewidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>980</width>
         <height>230</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rowCount">
        <number>7</number>
       </property>
       <property name="columnCount">
        <number>8</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_input</tabstop>
  <tabstop>select_input</tabstop>
  <tabstop>line_output</tabstop>
  <tabstop>select_output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exec_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>exec_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_input</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>input_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_output</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>output_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Form</sender>
   <signal>output_changed(QString)</signal>
   <receiver>line_output</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Form</sender>
   <signal>input_changed(QString)</signal>
   <receiver>line_input</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>input_modified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_output</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>output_modified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>input_changed(QString)</signal>
  <signal>output_changed(QString)</signal>
  <signal>x1_dim_changed(int)</signal>
  <signal>x2_dim_changed(int)</signal>
  <signal>x3_dim_changed(int)</signal>
  <signal>x1_deg_changed(int)</signal>
  <signal>x2_deg_changed(int)</signal>
  <signal>x3_deg_changed(int)</signal>
  <signal>type_cheb()</signal>
  <signal>type_lege()</signal>
  <signal>type_lagg()</signal>
  <signal>type_herm()</signal>
  <slot>input_clicked()</slot>
  <slot>output_clicked()</slot>
  <slot>samples_modified(int)</slot>
  <slot>dimension_modified(int)</slot>
  <slot>degree_modified(int)</slot>
  <slot>type_modified(bool)</slot>
  <slot>plot_clicked()</slot>
  <slot>exec_clicked()</slot>
  <slot>input_modified(QString)</slot>
  <slot>output_modified(QString)</slot>
  <slot>weights_modified(QString)</slot>
  <slot>lambda_calc_method_changed(bool)</slot>
  <slot>bruteforce_called()</slot>
  <slot>structure_changed(bool)</slot>
 </slots>
</ui>
